common/static/common/js/discussion/views/discussion_thread_view.js

```diff
@@ -280,6 +280,9 @@
                     responseCountFormat = ngettext(
                         '{numResponses} other response', '{numResponses} other responses', responseTotal
                     );
+                    if (responseTotal === 0) {
+                        this.$el.find('.response-count').hide();
```

Comment:
Does there need to be corresponding logic to show it again when a response is added? Or is the idea that it can only be the user's own changes so they won't mind it not showing?